parser/model/model.go

```diff
@@ -1164,8 +1164,9 @@ func (p PartitionType) String() string {
 
 // ExchangePartitionInfo provides exchange partition info.
 type ExchangePartitionInfo struct {
-	ExchangePartitionID    int64 `json:"exchange_partition_id"`
-	ExchangePartitionDefID int64 `json:"exchange_partition_def_id"`
+	// It is nt tableID when table which has the info is a partition table, else pt tableID.
+	ExchangePartitionTableID     int64 `json:"exchange_partition_table_id"`
+	ExchangePartitionPartitionID int64 `json:"exchange_partition_partition_id"`
```

Comment:
Same, I think you can keep `ExchangePartitionDefID` here
```suggestion
	ExchangePartitionDefID int64 `json:"exchange_partition_def_id"`
```